### ISSUE TITLE
Missing Test Case - 462. Minimum Moves to Equal Array Elements II

### ISSUE THREAD

**Issue by chiruplayer** [LeetCode-Feedback/LeetCode-Feedback#3568]
#### Your LeetCode username

Worakais

#### Category of the bug
- [ ] Question
- [ ] Solution
- [ ] Language
- [ *] Missing Test Cases 

#### Description of the bug

The constraints in the question are 

    n == nums.length
    1 <= nums.length <= 105
    -109 <= nums[i] <= 109

I used these constraints and python to create a custom test case . However when running the testcase in custom testcases, i get the following error
Line 97: Char 22: runtime error: signed integer overflow: 2103215260 + 55200287 cannot be represented in type 'int' (solution.cpp)

#### Code you used for Submit/Run operation

```
class Solution {
public:
    int minMoves2(vector<int>& nums) {
        unsigned long long int min_moves=INT_MAX;
        sort(nums.begin(),nums.end());
        int n = nums.size();
        vector<unsigned long long int> pre_sum(n);
        pre_sum[0]=nums[0];
        for(int i=1;i<n;i++){
            pre_sum[i]=pre_sum[i-1]+nums[i];
        }
        for(int i=0;i<n;i++){
            unsigned long long int part1=(unsigned long long int)(i+1)*nums[i] - pre_sum[i];
            
            unsigned long long int part2=(pre_sum[n-1]-pre_sum[i]) - (unsigned long long int)  (n-1-i)*nums[i] ;
            unsigned long long int total_moves = part1 + part2;
         
            min_moves = min(total_moves,min_moves);
        }
        return (int)min_moves;
        
    }
};
```

#### Language used for code

C++

#### Expected behavior

An int number must be given as output in "Expected answer"

#### Screenshots

![image](https://user-images.githubusercontent.com/8570787/118856934-1c866480-b8f5-11eb-9f69-82e5203ac4ee.png)

#### Additional context

This is the test case
[input3.txt](https://github.com/LeetCode-Feedback/LeetCode-Feedback/files/6510654/input3.txt)

**Comment by lc-kay** [comment]
Hi @chiruplayer 
Thank you for reaching out to us. I've relayed this issue to our team to investigate.

**Comment by lc-kay** [comment]
@chiruplayer This has been fixed with #3565 Thank you for your time.
